### DISCUSSION TITLE
Aggiornamenti

### DISCUSSION THREAD

**Discussion by luigirotondi** [iisgiua/giuaschool#799] [Richieste di aiuto]
Buongiorno @trinko gli aggiornamenti disponibili possono essere effettuati o bisogna attendere ancora?

**Comment by robertopadovesi** [comment]
Bisogna attendere finchè non sarà rilasciato un comunicato ufficiale. Come
specificato nei thread fissati, un aggiornamento al sistema senza
ufficialità, potrebbe mettere a "rischio" il corretto funzionamento
l'intero registro elettronico.

Il Dom 15 Dic 2024, 11:46 luigirotondi ***@***.***> ha
scritto:
